### ISSUE TITLE
Improve README.md

### ISSUE THREAD

**Issue by czgdp1807** [codezonediitj/pydatastructs#91]
#### Description of the problem

The README of the project should be updated. We need to replace, **Who are we?** and **How are we different?** with an **About** section which should contain the following words,
```
Currently, the aim of the project is to be a Python package for various data structures in computer science. In addition, we are also working on including parallel algorithms. To the best of our knowledge there doesn't exist any well designed library/package which has covered most of the data structures and algorithms including their parallel implementation. 

In future, i.e, after a few releases of the package when the software design will become stable, we also aim to provide APIs for the code in C++ and Java as well. For clarity, the interested students aren't expected to know C++ and Java. Knowledge of Python is enough for this project.
```

**Comment by yash2189** [comment]
Can I take this up as a contribution for gssoc'20?

**Comment by czgdp1807** [comment]
Hi, @yash2189 
There is already a PR open for this. You can go for, https://github.com/codezonediitj/pydatastructs/issues/96.

**Comment by czgdp1807** [comment]
Resolved by #93
